README.md

```diff
@@ -129,77 +100,123 @@ For some large and heavily trafficked sites, that case is rare. For others, it
     * SHA-256 hashes are long. Their hex representation would be 64 bytes, and we can base64 them to be ~42 (I think). We can't afford to send many hashes for both performance and privacy reasons.
-    * We could truncate those hashes, or use shorter ones, but then we'd run into potential correctness issues in case of collisions. We can run the math on that, to see if it's worth the risk.
     * The `sec-` prefix is there to ensure that requests are not attacker-generated.
-    * Ideally, we can get developers to pick a single resource to be the best dictionary for a certain path (where that path may only contain future versions of the same resource)
     * Any new resource as a dictionary for that path would override older ones. When sending requests, the browser would use the _most specific path_ for the request to get its dictionary.
 * When the server gets a request with the `sec-bikeshed-available-dictionary` header in it:
     * The server can simply ignore the dictionary if it doesn't have a diff that corresponds to said dictionary. In that case the server can serve the response without delta compression.
-    * If the server does have a corresponding diff, it can respond with that, indicating that as part of its `content-encoding` header, or some other header. There's no need to repeat the hash value, as there's only one.
+    * If the server does have a corresponding diff, it can respond with that, indicating that as part of its `content-encoding` header. There's no need to repeat the hash value, as there's only one.
       - For example, if we're using [shared brotli compression](https://datatracker.ietf.org/doc/draft-vandevenne-shared-brotli-format/), the `content-encoding: sbr` header can indicate that.
-    * In case the browser advertized a dictionary but then fails to successfuly fetch it from its cache *and* the dictionary was used by the server, the resource request should be terminated.
-
+    * In case the browser advertized a dictionary but then fails to successfuly fetch it from its cache *and* the dictionary was used by the server, the resource request should fail.
 
 ### Dynamic resources flow
 
-
-
-* Shared dictionary is downloaded out of band and declared using a dedicated header. That means that a dynamic resource's response would contains a `bikeshed-shared-dictionary-url` header, with the URL for the resource to be downloaded.
-    * The dictionary resource will be downloaded with CORS in “omit” mode, ensuring lack of credentials.
-    * It will be downloaded with “idle” priority, once the site is actually idle
+* Shared dictionary is downloaded out of band and declared using a `<link>` header or HTML tag with a `rel=bikeshed-dictionary` type and appropriate `as` for the destination it is to apply to.
+    * The dictionary resource will be downloaded with CORS in “omit” mode to discourage including user-specific private data in the dictionary.
```

Comment:
The only thing that worries me about making the language too strong on `omit` is that the real guarantee is provided by the CORS same-domain or access-control-allow mechanisms. Omit on the separate dictionary fetch is an enhancement to discourage including private information but it isn't the core of the protection.

Reply:
Sure. Maybe `The dictionary resource will be downloaded with CORS in “omit” mode to discourage including user-specific private data in the dictionary, since its data will be readable without credentials.`